templates/titian-mosaic.template

```diff
@@ -1,15 +1,15 @@
 ---
```

Comment:
Please revert the changes for descriptions. In most cases, the change in description has been made to make it shorter. The original description has been reviewed by our Tech writers, following the AWS writing guidelines.

Reply:
Corrected as requested.